contracts/HeapOrdering.sol

```diff
@@ -141,18 +141,22 @@ library HeapOrdering {
         Account memory initialAccount = getAccount(_heap, _rank);
         uint256 initialValue = initialAccount.value;
```

Comment:
not sure why it is called "initial".

Reply:
It's the account we start with, but I agree that it is not the best naming

Reply:
See #50